src/main/java/filtering_use_case/sortChoices.java

```diff
@@ -33,21 +33,21 @@ public ChoicesResponseModel select(ChoicesRequestModel requestModel){
 
             // Possibly have multi drop-down select?
             if((restaurant.getPriceBucket() == requestModel.getInputPriceBucket() ||
-                    restaurant.getPriceBucket() == 0)
+                    requestModel.getInputPriceBucket() == 0)
 
                     // Determines Location (Postal Code) and uses the first 3 letters for nearby rest
-                //&& (Objects.equals(restaurant.getLocation().substring(0, 3),
-                 //   requestModel.getInputLocation().substring(0, 3)) ||
-                 //   Objects.equals(restaurant.getLocation(), ""))
+                    //&& (Objects.equals(restaurant.getLocation().substring(0, 3),
+                    //   requestModel.getInputLocation().substring(0, 3)) ||
+                    //   Objects.equals(restaurant.getLocation(), ""))
 
                     // Can change to tags, would make cuisine type an input or keep the same but
                     // choices are limited
-                && (Objects.equals(restaurant.getCuisineType(), requestModel.getInputCuisineType()) ||
-                    Objects.equals(restaurant.getCuisineType(), ""))
+                    && (Objects.equals(restaurant.getCuisineType(), requestModel.getInputCuisineType()) ||
+                    Objects.equals(requestModel.getInputCuisineType(), "No Preference"))
```

Comment:
Maybe make the "No Preference" a static variable so that any screen that needs it doesn't have to copy paste this exact string

Reply:
alright